server/search/searchHelpers.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  transformToSearchDisplay: (movieObj) => {
+    const finalSearchResponse = movieObj.history.map((movie) => {
+      const finalObj = {}
+      const finalProviderArr = []
+      // console.log(movie)
+      
+
+
+      finalObj.title = movie.title
+      finalObj.imgUrl = movie.imgUrl
+      finalObj.providers = [movie.hulu ? movie.hulu: null, movie.disney ? movie.disney: null, movie.netflix ? movie.netflix : null, movie.hbo ? movie.hbo: null, movie.apple ? movie.apple : null, movie.amazon ? movie.amazon : null]
```

Comment:
I was JUST thinking to myself yesterday about how I couldn't remember how to use ternary operators when there are more than 2 options. This solves it!

Reply:
This could be `[movie.hulu || null, movie.disney || null, ...` but why have empty indexes in the array at all? Have you considered a different structure for the provider data?

Reply:
was late in night and thats the way i had the schema set up for the database.  I wanted to just return the ones that were necessary but yeah... didnt want to waste time diving into a nested data structure when i could just place null and have something else rendered if property comes back null